daq_utils.py

```diff
@@ -84,7 +84,7 @@ def init_environment():
   varname = "DETECTOR_OFFLINE"
   if varname in os.environ:
     detectorOffline = int(os.environ[varname])
-
+  setBlConfig("beamCheck",1)
```

Comment:
this needs to be using the config_param string

Reply:
daq_utils required importing config_param, but all references to beamCheck have been replaced with the config_param variable

Reply:
e8939d5a32f49f708c7ebd6833f80cadc44e8ba4